.gitignore

```diff
@@ -2,6 +2,6 @@ UmbracoCms.8.1.1/*
 /src/.vs
 /src/*/obj/*
 /src/*/bin/*
-/src/packages
+/src/**/*.pubxml.user/src/packages
```

Comment:
Is this to exclude the .pubxml.user file types?
If so lets simplify it, call me to discuss

Reply:
It seems the newline striped and /src/packages is duplicated. will talk about it to confirm my understanding